samples/rendezvous/src/main/main.html

```diff
@@ -8,21 +8,43 @@ <h1>EVENTS</h1>
         </dv.link>
       </div>
 
+      <dv.choose chooseSelectPlaceholder="Filter"
```

Comment:
We should try to keep the code consistent. I think that it's better to indent inputs by 2 spaces. That is, instead of starting `i2` where `i1` starts:
```
<concept.elem i1=v1
              i2=v2
              i3=v3 />
```

Start it after 2 spaces of where `concept.elem` starts:
```
<concept.elem
   i1=v1
   i2=v2
   i3=v3 />
```

because expressions can be long.

Also, we should put the closing tag symbol (`>` or `/>`) together with the last line. Instead of this:
```
<dv.if condition=foo
     && bar
     && baz>
```

I think we should do this:

```
<dv.if condition=foo
     && bar
     && baz>
```

I don't think one format is necessarily better than the other one, but we should choose one and apply it consistently.

Reply:
Thank you for explaining! I changed all of the ones I found!